code-style/licenseHeader.txt

```diff
@@ -1,5 +1,5 @@
 ^(/\*|#)$
-^(\s\*|#) Copyright (20\d\d)(-20\d\d)?
+^(\s\*|#) Copyright (20\d\d)(-20\d\d)? Crown Copyright$
```

Comment:
We deliberately don't require 'Crown Copyright' to allow external contributions such as: https://github.com/gchq/gaffer-docker/blob/master/Dockerfile.build and https://github.com/gchq/gaffer-docker/blob/master/resources/standalone.xml